src/mgr/DaemonHealthMetricCollector.cc

```diff
@@ -2,100 +2,100 @@
 
 #include "include/health.h"
 #include "include/types.h"
-#include "OSDHealthMetricCollector.h"
+#include "DaemonHealthMetricCollector.h"
 
 
 
 ostream& operator<<(ostream& os,
-                    const OSDHealthMetricCollector::DaemonKey& daemon) {
+                    const DaemonHealthMetricCollector::DaemonKey& daemon) {
   return os << daemon.first << "." << daemon.second;
 }
 
 namespace {
 
-class SlowOps final : public OSDHealthMetricCollector {
-  bool _is_relevant(osd_metric type) const override {
-    return type == osd_metric::SLOW_OPS;
+class SlowOps final : public DaemonHealthMetricCollector {
+  bool _is_relevant(daemon_metric type) const override {
+    return type == daemon_metric::SLOW_OPS;
   }
   health_check_t& _get_check(health_check_map_t& cm) const override {
     return cm.get_or_add("SLOW_OPS", HEALTH_WARN, "");
   }
-  bool _update(const DaemonKey& osd,
-               const OSDHealthMetric& metric) override {
+  bool _update(const DaemonKey& daemon,
+               const DaemonHealthMetric& metric) override {
     auto num_slow = metric.get_n1();
     auto blocked_time = metric.get_n2();
     value.n1 += num_slow;
     value.n2 = std::max(value.n2, blocked_time);
     if (num_slow || blocked_time) {
-      osds.push_back(osd);
+      daemons.push_back(daemon);
       return true;
     } else {
       return false;
     }
   }
   void _summarize(health_check_t& check) const override {
-    if (osds.empty()) {
+    if (daemons.empty()) {
       return;
     }
     static const char* fmt = "%1% slow ops, oldest one blocked for %2% sec";
     check.summary = boost::str(boost::format(fmt) % value.n1 % value.n2);
     ostringstream ss;
-    if (osds.size() > 1) {
-      ss << "osds " << osds << " have slow ops.";
+    if (daemons.size() > 1) {
+      ss << "daemons " << daemons << " have slow ops.";
     } else {
-      ss << osds.front() << " has slow ops";
+      ss << daemons.front() << " has slow ops";
     }
     check.detail.push_back(ss.str());
   }
-  vector<DaemonKey> osds;
+  vector<DaemonKey> daemons;
 };
 
 
-class PendingPGs final : public OSDHealthMetricCollector {
-  bool _is_relevant(osd_metric type) const override {
-    return type == osd_metric::PENDING_CREATING_PGS;
+class PendingPGs final : public DaemonHealthMetricCollector {
+  bool _is_relevant(daemon_metric type) const override {
+    return type == daemon_metric::PENDING_CREATING_PGS;
   }
   health_check_t& _get_check(health_check_map_t& cm) const override {
     return cm.get_or_add("PENDING_CREATING_PGS", HEALTH_WARN, "");
   }
-  bool _update(const DaemonKey& osd,
-               const OSDHealthMetric& metric) override {
+  bool _update(const DaemonKey& daemon,
```

Comment:
i think "osd" still applies here. as only osd sends PENDING_CREATING_PGS metrics to mgr.